### ISSUE TITLE
Change button copy on tribe homepage (not-logged in) to Become member for public tribes

### ISSUE THREAD

**Issue by ralphboeije** [part-up/part-up#1178]
- Open an inprivate browser
- Go to a public tribe homepage: e.g. https://part-up.com/tribes/partup-coop
- See that the center button now has the "Request access" copy

- Open a browser and log in to part-up
- Go to a public tribe homepage your account is not yet member of
- See that the center button now has the "Become member" copy

The latter is also requested for not-logged in users

**Comment by Seagullible** [comment]
Hello, I'd like to pick this issue up. I followed the directions to replicate the issue - after a user is logged in, the button says "Join tribe" instead of "Become member." It sounds like you want the "Request access" button text to be changed to "Join tribe". Am I understanding that correctly?

**Comment by ralphboeije** [comment]
@Seagullible yes you are, and your are very welcome to pick this issue up. Please do not use hardcopy text, as there are i18n keys for that in the phraseapp json.

**Comment by Seagullible** [comment]
Sorry, I don't think I'll be able to complete this issue after all. I'm having trouble figuring out how to test it.

**Comment by ralphboeije** [comment]
Hi @Seagullible you can always test it on master.staging.part-up.com with the fixture data. If you need any specific help, please ask us on the tribe chat: https://part-up.com/tribes/development/chat
